modules/ROOT/pages/configure-monitoring-cloudhub.adoc

```diff
@@ -7,11 +7,15 @@ To start monitoring Mule apps that are managed in CloudHub, you must deploy them
 
 == Supported Versions of Mule Deployed to CloudHub
 
-Anypoint Monitoring images (for example, 3.8.7-AM) receive the latest patch updates once per month, while the non-AM images receive patch updates as soon as they are available. If your production environment requires a Mule runtime engine patch, you should switch to a non-AM runtime.
+Anypoint Monitoring non-AM images receive patch updates as soon as they are available, while -AM images receive the latest patch updates only once per month. It is recommended that you switch to a non-AM image if your production environment requires a Mule runtime engine patch.
```

Comment:
Are we still saying "Mule Runtime" I thought we were saying "Mule" or "Mule Runtime Engine" (not sure of the capitalization the last term instead.

Reply:
It says Mule runtime engine.   That's the correct capitalization.